packages/preview/tictactoe/1.0.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "tictactoe"
```

Comment:
The name of the package is canonical, can you please find another name? For example, a similar package that implements a Tetris game is called `soviet-matrix`, not `tetris`.